qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/settings/settings_model.py

```
@@ -8,40 +8,45 @@
 from mantidqtinterfaces.Engineering.gui.engineering_diffraction.tabs.common import output_settings
 from os import path
 from mantid.kernel import logger
-from numpy import all, array, concatenate, abs, eye
+from numpy import all, array, concatenate, abs, eye, ndarray
```

Ruff (0.16.1)

[error] Import `all` is shadowing a Python builtin

(A004)